Documentation/flexvolume.md

```diff
@@ -46,7 +46,36 @@ Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path
 Rancher provides an easy way to configure kubelet. The FlexVolume flag will be shown later on in the [configuring the FlexVolume path](#configuring-the-flexvolume-path).
 It can be provided to the kubelet configuration template at deployment time or by using the `up to date` feature if Kubernetes is already deployed.
 
-Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
+Rancher deploys kubelet as a docker container, you need to mount the host's flexvolume path into the kubelet image as a volume,
+this can be done in the extra_binds section of the kubelet cluster config.
+
+* Configure the rancher deployed kubelet
```

Comment:
I think removing the bullet point list here and formulating it into a "flow" of text like other Rook docs, makes it look more unified.